var/spack/repos/builtin/packages/py-spython/package.py

```diff
@@ -28,4 +29,4 @@ class PySpython(PythonPackage):
 
     depends_on("py-setuptools", type="build")
 
-    depends_on("py-semver@2.8.1:", type=("build", "run"))
+    depends_on("py-semver@2.8.1:", when="@:0.2.14", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("py-semver@2.8.1:", when="@:0.2", type=("build", "run"))
```
Just in case a critical bug is found and you decide to make a 0.2.15 release someday.